### PR TITLE
Renamed event and fix firing of o-chatter-rendered

### PR DESCRIPTION
**Pull request by xdu-odoo**

### CODE REVIEW

File: addons/mail/static/src/owl/components/chatter/chatter.js [hunk 1]
```diff
@@ -22,22 +22,43 @@ class Chatter extends Component {
             const chatter = state.chatters[props.chatterLocalId];
             const threadLocalId = chatter.threadLocalId;
             const thread = threadLocalId ? state.threads[threadLocalId] : undefined;
+            let attachments;
             if (thread) {
-                const attachments = thread.attachmentLocalIds.map(attachmentLocalId =>
+                attachments = thread.attachmentLocalIds.map(attachmentLocalId =>
```
Comment (alexkuhn) [inline addons/mail/static/src/owl/components/chatter/chatter.js:27]: missing `compareDepth: { attachments: 1 }`

**Review by alexkuhn — COMMENTED** [review]
Don't really get what the issue it tries to "fix".
Just see that it used `mounted`/`patched` instead of `useStore`'s selector, but I think it's almost like before, no?

**Comment by xdu-odoo** [comment]
Yes, this part just moves the event firing to mounted and patched as it's better (following Seb)

File: addons/mail/static/src/owl/components/chatter/chatter.js (hunk 1 above)
Comment (xdu-odoo) [reply addons/mail/static/src/owl/components/chatter/chatter.js:27]: As second argument of map ? I don't know that one .. ?

Comment (alexkuhn) [reply addons/mail/static/src/owl/components/chatter/chatter.js:27]: this is an option of `useStore(selector, options)`, so `useStore(/* selector */, { compareDepth: { attachments: 1 } })` (ofc multi-line for scoring style points)

Comment (xdu-odoo) [reply addons/mail/static/src/owl/components/chatter/chatter.js:27]: Okay didn't know about that !

**Comment by xdu-odoo** [comment]
Red because of migrations

**Comment by xdu-odoo** [comment]
Green :+1:

**Comment by xdu-odoo** [comment]
Rebased and green, waiting to be merged :+1:

**Review by alexkuhn — APPROVED** [review]

**Pull request merged**